External/AvalonTools/AvalonToolkit_1.1_beta_patch.txt

```diff
@@ -0,0 +1,165 @@
+*** ./SourceDistribution/common/reaccsio.c.orig	2016-01-31 19:46:03.709178428 +0000
```

Comment:
Yes, I had noticed that there was a newer version. I sticked to the previous one because, as the README explicitly mentioned that "the version is important", I thought there was some some other reason to still use 1.1beta. I check if the patch is still needed and eventually update it to 1.2.